streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StateStoreNode.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.streams.kstream.internals.graph;
+
+import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder;
+import org.apache.kafka.streams.state.StoreBuilder;
+
+public class StateStoreNode extends StreamsGraphNode {
+
+    protected final StoreBuilder storeBuilder;
+
+    public StateStoreNode(final StoreBuilder storeBuilder) {
+        super(storeBuilder.toString(), false);
```

Comment:
@bbejeck Could you confirm whether the intention is to pass `storeBuilder.name()` [or] `storeBuilder.toString()` ?

Reply:
Good catch. I think you're right.

Reply:
oh, yeah, it's not a bug, just a little confusing. I neglected to say that.